packages/hooks/README.md

```diff
@@ -1,34 +1,35 @@
 # WP-JS-Hooks
 
-A lightweight & efficient EventManager for JavaScript in WordPress.
-
+A lightweight & efficient event manager.
 
 ### API Usage
-API functions can be called via the global `wp.hooks` like this `wp.hooks.addAction()`, etc.
-
-* `addAction( 'hook', 'vendor/plugin/function', callback, priority )`
-* `addFilter( 'hook', 'vendor/plugin/function', callback, priority )`
-* `removeAction( 'hook', 'vendor/plugin/function' )`
-* `removeFilter( 'hook',  'vendor/plugin/function' )`
-* `removeAllActions( 'hook' )`
-* `removeAllFilters( 'hook' )`
-* `doAction( 'hook', arg1, arg2, moreArgs, finalArg )`
-* `applyFilters( 'hook', content, arg1, arg2, moreArgs, finalArg )`
-* `doingAction( 'hook' )`
-* `doingFilter( 'hook' )`
-* `didAction( 'hook' )`
-* `didFilter( 'hook' )`
-* `hasAction( 'hook' )`
-* `hasFilter( 'hook' )`
 
+* `addAction( 'hookName', 'functionName', callback, priority )`
+* `addFilter( 'hookName', 'functionName', callback, priority )`
+* `removeAction( 'hookName', 'functionName' )`
+* `removeFilter( 'hookName',  'functionName' )`
+* `removeAllActions( 'hookName' )`
+* `removeAllFilters( 'hookName' )`
+* `doAction( 'hookName', arg1, arg2, moreArgs, finalArg )`
+* `applyFilters( 'hookName', content, arg1, arg2, moreArgs, finalArg )`
+* `doingAction( 'hookName' )`
+* `doingFilter( 'hookName' )`
+* `didAction( 'hookName' )`
+* `didFilter( 'hookName' )`
+* `hasAction( 'hookName' )`
+* `hasFilter( 'hookName' )`
 
-### Background
-See ticket [#21170](http://core.trac.wordpress.org/ticket/21170) for more information.
+Hooks can be added to an object via composition:
+
+`myObject.hooks = new Hooks( myObject )`
```

Comment:
We should update the docs after the recent changes

Reply:
updated in https://github.com/WordPress/packages/pull/40/commits/f1c4f1fa0d8760e72d93604292e3466b9d2716c5